internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go

```diff
@@ -102,6 +102,23 @@ docker-build: ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
```

Comment:
I guess we need test data as we updated the plugins.

Reply:
+1, testdata/ and docs would be helpful to users (as this was one of the frequently asked options by upstream community)

Reply:
+1

Reply:
testdata looks good to me.